feat: adding coindesk bitcoin data (#10)
influxdata/influxdb2-sample-data: bitcoin-price-data/README.md

# CoinDesk Bitcoin Price Data

> “Powered by CoinDesk” - https://www.coindesk.com/price/bitcoin

Reviewer: ```suggestion
_[Powered by CoinDesk](https://www.coindesk.com/price/bitcoin)_
```

This directory contains the following files generated from the CoinDesk Bitcoin Pricing Data (https://www.coindesk.com/price/bitcoin).

### bitcoin-currentprice-annotated.csv

> This is the latest Bitcoin pricing data from the API in annotated CSV format. This is meant to be imported into InfluxDB via the `csv.from` function.

Reviewer: ```suggestion
Latest Bitcoin pricing data from the CoinDesk API in annotated CSV format. Use the `csv.from` function to import the data into InfluxDB.
```

### bitcoin-historical-annotated.csv

> This is the last 30 days of Bitcoin pricing data from the API in annotated CSV format. This is meant to be imported into InfluxDB via the `csv.from` function.

Reviewer: ```suggestion
Last 30 days of Bitcoin pricing data from the CoinDesk API in annotated CSV format. Use the `csv.from` function to import the data into InfluxDB.
```

### currentprice.json

> This is the latest Bitcoin pricing data from the API in JSON format.

Reviewer: ```suggestion
Latest Bitcoin pricing data from the CoinDesk API in JSON format.
```

### currentprice.lp

> This is the latest Bitcoin pricing data from the API in line protocol format. This is meant to be imported into InfluxDB via the web UI or CLI `influx write` command.

Reviewer: ```suggestion
Latest Bitcoin pricing data from the CoinDesk API in line protocol format. Use the `csv.from` function to import the data into InfluxDB.
```
